ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_approved-decision.js

```diff
@@ -0,0 +1,175 @@
+/// <reference types ='Cypress'/>
+
+// uri to be updated once academisation API is integrated
+let url = Cypress.env('url') + '/task-list/2050?rd=true'
+
+describe('103195 Approved record decision', ()=> {
+    
+    beforeEach(() => {
+        cy.visit(url)
+        cy.clearCookies()
+
+    })
+    // Approve Happy Path journey
+    // *** A new project is need to run tests - skipping for now
+    it.skip('TC01: J1 - Navigate to project to record a decision', () => {
+        cy.get('[id="record-decision-link"]').click()
+        cy.get('[id="approved-radio"]').click()
+    })
+
+    // Edit Approval Path - Regional Director, No/Yes conditions set 
+    it('TC02: J1 Edit a recorded decision Approval - Reg Director Region, No Conditions', () => {
```

Comment:
For my understanding what does TC stand for?

Reply:
TC = Test case